text/0000-temp-config-settings.md

```diff
@@ -13,65 +13,368 @@ nit-picking the "gotchas" in code examples. :heart:
+is not something that is available in any Pando package.
 
 # Motivation
 
-Why are we doing this? What use cases does it support? What is the expected
-outcome?
+The motivation behind this RFC is to allow both the Headless-styles and React
+libs to be more flexible and configurable for future use-cases by leveraging the
+power of Panda CSS (i.e. Panda is basically what HS end-goal was to be).
+
+## Why are we doing this?
+
+There is an issue in our Company FE tech (big "C") regarding the lack of structure
+and consistency of tooling (especially with styling). This has led to unnecessary
+chaos and potential bugs in our codebase. For example, the home page of
+Pluralsight.com is loading 1.5MB of CSS to the client. Likewise, it's also using
+mutliple style libraries.
+
+From a basic check of the library page (single page), it looks like we are using the following
+style libraries:
+
+- Bootstrap
+- Tailwind
+- Glamor
+- Styled-components/CSS Modules
+- Headless-styles
+
+We need to solve this problem. :scream:
+
+## What use cases does it support?
 
-Please focus on explaining the motivation so that if this RFC is not accepted,
-the motivation could be used to develop alternative solutions. In other words,
-enumerate the constraints you are trying to solve without coupling them too
-closely to the solution you have in mind.
+This RFC will allow us to support the following use cases:
+
+1. Teams that need more flexibility and configurability than what Pando
+currently provides.
+2. Teams that prefer to use a CSS-in-JS solution.
+3. Teams that prefer _not_ to use a CSS-in-JS solution.
+4. Teams that use a client-side framework (any library).
+5. Teams that use a server-side framework (any library).
+
+## What is the expected outcome?
+
+The expected outcome is that we will have a more flexible and configurable
+solution in Pando for teams to use while creating structure and standardization
+around the style tooling that we use across the company. In result, this should
+lead to a more performant and consistent experience for our customers and developers.
+
+Additionally, it should help team members be more productive and efficient by
+reducing the amount of time they spend on styling and debugging styling issues.
+This will also allow a more successful transition for teammates that are moving
+from one team to another (regardless of the product).
+
+...or...a major brand update. :wink:
 
 # Detailed design
 
-This is the bulk of the RFC. Explain the design in enough detail for somebody
-familiar with a Pando library to understand, and for somebody familiar with the
-implementation to implement. This should get into specifics and corner-cases,
-and include examples of how the feature is used. Any new terminology should be
-defined here.
+The following design assumes you are familiar with the
+[Core Features and Concepts](https://panda-css.com/docs/concepts/cascade-layers) of Panda CSS.
```

Comment:
I think it's probably fair to assume that most people reading this aren't familiar with Panda CSS, so having the main high-level (bullet point) killer features mentioned here somehow would be helpful for those not familiar with Panda. For example, that link just explains cascade layers which, while super useful, are a CSS feature, not specific to Panda CSS.